### ISSUE TITLE
At-a-glance reports are empty

### ISSUE THREAD

**Issue by lars-t-hansen** [NAICNO/Jobanalyzer#273]
After landing #269, we're now getting empty at-a-glance reports.

**Comment by lars-t-hansen** [comment]
webload-5m-and-upload.sh fails:
```
NAICREPORT FAILED
While unmarshaling output of /itf-fi-ml/home/larstha/sonar/sonalyze [load --half-hourly --fmt=json,host,rcpu,rmem,rgpu,rgpumem --config-file /itf-fi-ml/home/larstha/sonar/ml-nodes.json --from 1d --data-path /itf-fi-ml/home/larstha/sonar/data]
json: cannot unmarshal object into Go value of type []*glance.sonalyzeRawLoadData
```

**Comment by lars-t-hansen** [comment]
I say again, #270 ...

**Comment by lars-t-hansen** [comment]
Also, #233.

**Comment by lars-t-hansen** [comment]
Fixed by deploying 3fff1d7357d0839125907c907759a564e4ec84c6.
